Slack.Automation/Promact.Core.Repository/ScrumRepository/ScrumBotRepository.cs

```diff
@@ -383,7 +377,51 @@ public class ScrumBotRepository : IScrumBotRepository
         }
 
 
-
+        /// <summary>
+        /// Used to add channel manually by command "add channel channelname"
+        /// </summary>
+        /// <param name="ChannelName"></param>
+        /// <param name="ChannelId"></param>
+        /// <param name="Username"></param>
+        /// <returns></returns>
+        private async Task<string> AddChannelManually(string ChannelName, string Username, string ChannelId)
+        {
+            var returnMsg = string.Empty;
+            if (ChannelId.StartsWith(StringConstant.GroupNameStartsWith, StringComparison.Ordinal))
```

Comment:
To check just whether the channelId starts with "G", whether the channel is a private channel or not. If it is private channel then  channelId will start with "G".  In this way manual addition of public channel and direct messages are not allowed.